v3/util/oid.go

```diff
@@ -49,6 +49,8 @@ var (
 	SubjectInfoAccessOID    = asn1.ObjectIdentifier{1, 3, 6, 1, 5, 5, 7, 1, 11}       // Subject Info Access Syntax
 	SubjectKeyIdentityOID   = asn1.ObjectIdentifier{2, 5, 29, 14}                     // Subject Key Identifier
 	ReasonCodeOID           = asn1.ObjectIdentifier{2, 5, 29, 21}                     // CRL Reason Code
+	AdobeTimestampOID       = asn1.ObjectIdentifier{1, 2, 840, 113583, 1, 1, 9, 1}    // Adobe Timestamp x509 extension
+	AdobeArchiveRevInfoOID  = asn1.ObjectIdentifier{1, 2, 840, 113583, 1, 1, 9, 2}    // Adobe Archive Revocation Info x509 extension
```

Comment:
Minor: It looks like each commented section of OIDs is sorted, so these should probably be at the top of this list.

Reply:
will do